tests/cli/superset/main_test.py

```diff
@@ -161,4 +162,4 @@ def test_superset_jwt_auth(mocker: MockerFixture) -> None:
         catch_exceptions=False,
     )
 
-    JWTAuth.assert_called_with("SECRET")
+    JWTAuth.assert_called_with("SECRET", URL("http://localhost:8088/"))
```

Comment:
baseurl is better suffix without `/`. Then you can prefix `api/v1/security/csrf_token/` with `/`

Reply:
thanks, @zephyring! The `baseurl` wasn't included in the constructor before, so I just copied what was sent to the `runner.invoke` [here](https://github.com/preset-io/backend-sdk/blob/main/tests/cli/superset/main_test.py#L158):
``` python
runner.invoke(
  superset_cli,
  ["--jwt-token=SECRET", "http://localhost:8088/", "export"],
  catch_exceptions=False,
)
```

The last trailing slash is included in other tests (like [here](https://github.com/preset-io/backend-sdk/blob/main/tests/auth/password_test.py#L23) and [here](https://github.com/preset-io/backend-sdk/blob/main/tests/auth/password_test.py#L48)), do you think this should be updated in other places as well? 

I performed a local test using both `http://localhost:8088/` and `http://localhost:8088` in the CLI command and printed the endpoint -- both scenarios produced the same URL (`http://localhost:8088/api/v1/security/csrf_token/`) and the command worked properly. 

Let me know your thoughts. Thank you!

Reply:
It's a convention to have base_url ending without `/`. Not sure the other examples syntax for string / string is a good pattern to follow. maybe @betodealmeida ?

Reply:
It should work either way here, we're using [yarl](https://pypi.org/project/yarl/) for URL composition and it should do the right thing.